pom.xml

```diff
@@ -79,7 +79,7 @@
     <maven.compiler.source>8</maven.compiler.source>
     <!-- Supress checkstyle errors on legacy com.iplanet and com.sun.identity packages -->
     <checkstyleUnitTestSuppressionsLocation>checkstyle/suppressions.xml</checkstyleUnitTestSuppressionsLocation>
-    <opendj.version>4.9.4</opendj.version>
+    <opendj.version>4.9.5-SNAPSHOT</opendj.version>
```

Comment:
This is one of the weird things I haven't understood yet. If the products bring in consistently the same commons BOM, you get better separation of responsibilities. (Right now IG ships with DJ, because openig-saml brings in openam-shared that depends on opendj-core/grizzly for some reason?) Also it would mean that patching commons wouldn't need additional changes in opendj as well.